.github/workflows/playwright.yml

```
@@ -0,0 +1,32 @@
+name: Playwright Tests
+on:
+  pull_request: {}
+  push:
+    branches: [livekit, full-mesh]
+jobs:
+  test:
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
```

To match the other workflows:

```suggestion
          cache: "yarn"
          node-version-file: ".node-version"
```